src/Microsoft.Performance.SDK.Runtime/ColumnBuilding/Processors/BuiltColumnVariantReflector.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.Performance.SDK.Runtime.ColumnVariants.TreeNodes;
+
+namespace Microsoft.Performance.SDK.Runtime.ColumnBuilding.Processors;
+
+/// <summary>
+///     Exposes the <see cref="IColumnVariantsTreeNode"/> that was passed to the processor.
+/// </summary>
+internal class BuiltColumnVariantReflector
+    : IColumnVariantsProcessor
```

Comment:
could we either `sealed` or struct?

Reply:
Made this sealed class since making it a struct breaks fetching the output since this is passed to the builders